### PR TITLE
feat: modified registy update command

### LINKED ISSUE #87: fix: input field are empty for update registry
**Problem**

When updating the registry, all the input field are empty.
If the user gets the value in the input field like in web UI, It is more easier to make changes to them.

**Solution**

Values need to shown while using update registry command

### PR DESCRIPTION
**Pull request by Althaf66**
This fixes #87 

Registry values where shown in the input field so, user have easier to update the values in the registry. This fuctionality present in the Web UI

### CODE REVIEW

File: cmd/harbor/root/registry/update.go [hunk 1]
```diff
@@ -7,82 +7,67 @@ import (
...
-
+			update.UpdateRegistryView(updateView)
+			err = runUpdateRegistry(updateView, registryId)
 			if err != nil {
 				log.Errorf("failed to update registry: %v", err)
 			}
 		},
 	}
 
-	flags := cmd.Flags()
```
Comment (bupd) [inline cmd/harbor/root/registry/update.go:63]: Thanks for your contribution! Could you please add back the below flags? They're crucial for  automation and scripting processes, and their absence could potentially disrupt CI/CD pipelines.

Thanks.

**Review by bupd — CHANGES_REQUESTED** [review]

File: cmd/harbor/root/registry/update.go (hunk 1 above)
Comment (Althaf66) [reply cmd/harbor/root/registry/update.go:63]: Yes. I will add those flags.

**Comment by amands98** [comment]
heads up: merge conflicts

**Comment by bupd** [comment]
@Althaf66 merge conflicts.

otherwise, LGTM 🙌

**Comment by Vad1mo** [comment]
update with empty id and secret shoould work

**Review by Vad1mo — APPROVED** [review]

**Pull request merged**